.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,11 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Bolt.new Help Center
+    url: https://support.bolt.new
+    about: Official central repository for tips, tricks, tutorials, known issues, and best practices for bolt.new usage.
+  - name: Billing Issues
+    url: https://support.bolt.new/Billing-13fd971055d680ebb393cb80973710b6
+    about: Instructions for billing and subscription related support
+  - name: Discord Chat
+    url: https://discord.gg/stackblitz
+    about: Ask questions and discuss with other Bolt users in real time.
```

@patak-dev is this what you suggested?

```suggestion
    about: Build, share, and learn with the other Bolters in real time.
```